Remove timeout in async_test for html tests

qiuzhong opened the pull request:
Remove all the timeout in `async_test` for html tests.
Affected tests: 125
Before: Pass: 64, Failed: 61
After:  Pass: 64, Failed: 61
Related: #11120

qiuzhong committed:
Remove timeout in async_test for html tests

Remove all the timeout in `async_test` for html tests.
Affected tests: 125
Before: Pass: 64, Failed: 61
After:  Pass: 64, Failed: 61
Related: #11120

jdm commented:
I saw those "no output" errors on several of the PRs removing the test-specific timeout values.

jgraham commented:
It happens more now because we filtered out a lot of the process output from Firefox in order to avoid overflowing the logs, but the flip side is we experience more timeouts. If we wanted to fix it we'd presumably have to arrange for a periodic ping to be logged.

foolip commented:
With https://github.com/web-platform-tests/wpt/pull/14096 merged it should be possible to just rebase this or otherwise retrigger checks.

qiuzhong commented:
Found lots of errors like this:

```
Traceback (most recent call last):
  File "/home/test/web-platform-tests/tools/wptserve/wptserve/handlers.py", line 310, in __call__
    rv = self.func(request, response)
  File "/home/test/web-platform-tests/html/semantics/embedded-content/media-elements/track/track-element/cors/support/cors-tester.py", line 36, in main
    line = 'cors = ' + cors + ' | cookie = ' + cookie.value;
AttributeError: 'str' object has no attribute 'value'
```

foolip commented on html/browsers/browsing-the-web/history-traversal/browsing_context_name_cross_origin_2.html:
```diff
@@ -6,7 +6,7 @@
 <iframe id="test"></iframe>
 <script>
-var t = async_test(undefined, {timeout:10000});
+var t = async_test(undefined);
```
Can you also remove the undefined arguments? They were only there to make it possible to pass in the second argument.

qiuzhong committed:
Remove unnecessary undefined parameter for async_test

After remove the timeout parameter from async_test, the first parameter
`undefined` is no use and should be removed.

foolip commented:
Thanks @qiuzhong, one down, a few more PRs to go :)

foolip commented:
https://tools.taskcluster.net/groups/DWgMz-OFQOKp5iktSe1gSg unfortunately failed so a clean before/after diff view for this PR isn't possible.

foolip commented:
Filed https://github.com/web-platform-tests/wpt/issues/14207 for the fact that Taskcluster is failing. Once there's another working run I'll compare, but then more changes will be lumped in.

foolip commented:
That's been resolved, here are diff views that includes the changes in this PR and more:
https://wpt.fyi/results/html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff

Just one seeming regression:
https://wpt.fyi/results/html/semantics/embedded-content/the-iframe-element/iframe_sandbox_popups_escaping-1.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff

Not quite regressions, see https://github.com/web-platform-tests/wpt.fyi/issues/796:
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-classic.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-module.html?product=chrome%4039e6009b92&product=chrome%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-classic.html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff
https://wpt.fyi/results/html/semantics/scripting-1/the-script-element/module/dynamic-import/no-active-script-manual-module.html?product=firefox%4039e6009b92&product=firefox%405438f72ea0&diff

foolip commented:
Ah, but iframe_sandbox_popups_escaping-1.html wasn't modified, so that's probably a flaky test.
